### PR TITLE
Every donator request. Finally.

### PR DESCRIPTION
**Pull request by shayoki**
## About The Pull Request

This PR is meant to add toys, clothes, markings and more cosmetics requested by donators of Bubberstation.
This was a task I assigned myself to take on for a long while. I decided to push this out in a PR to track progress.

Artists: Crumpaloo, Casey/Keil, Wolf751, bitsynapse.

<details>
<summary>Full list of additions</summary>
<br> -Blovy's plush, bottle, and suit. ✓
<br> -Farren's markings, hair, tail, wings, more. ✧ (semi-complete. but mostly good to go.)
<br> -Astroturf's plush. ✓
<br> -exfil's wings. ✓
<br> -patriot210's clothing set ✓
<br> -detrender's in-hand item ✓
<br> -ironknight060's clothing set and item. ✓
<br> -Gavla's birthday plush ✓
<br> -Big deer hand sprites ✓
<br> -Andrew Mower's plush ✓
<br> -Ringing bell collars (needs more modularization) ✓
<br>
<br>
<br>ignore the commits from May 1, 2024. I am bad at github.
</details>

## Why It's Good For The Game

Supporters of the server should receive in-game things as thanks.

## Proof Of Testing

Every commit is tested on my local beforehand before it is pushed. 

https://github.com/Bubberstation/Bubberstation/assets/96078776/833dbf4b-b63b-4205-8299-65ec7f170e81

https://github.com/Bubberstation/Bubberstation/assets/96078776/48ba3d2d-24ff-4535-b7d4-4b4f644627f2

https://github.com/Bubberstation/Bubberstation/assets/96078776/2b4ba30c-d7f4-4b6c-92d2-dcf5df7c78b2

image (6)

## Changelog

:cl: Shayoki, Crumpaloo, Keila, Wolf751, bitsynapse.
add: Donator toys and clothes have arrived.

### CODE REVIEW

**Comment by projectkepler-ru** [comment]
Can you do andrew mower's plushie too

**Comment by theselfish** [comment]
I'd make my own PR. But you're doing more to these files that I would.
image
So if you could do this too, I'd be thankful.
Literally just //.

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_toys.dm [hunk 1]
```diff
@@ -48,3 +48,20 @@
 	name = "Carrot Bag"
 	item_path = /obj/item/sbeacondrop/carrot
 	ckeywhitelist = list("slippyjoe")
```
Comment (theselfish) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_toys.dm:50]:
```suggestion
	//ckeywhitelist = list("slippyjoe")
```

As said in my other comment. Saves everyone time.

**Comment by projectkepler-ru** [comment]
if everything works hit me up again and we'll merge

**Review by projectkepler-ru — APPROVED** [review]

**Review by Majkl-J — APPROVED** [review]

**Pull request merged**